conversion/conversion.go

```diff
@@ -582,7 +582,7 @@ func UpdateDDLForeignKeys(ctx context.Context, adminClient *database.DatabaseAdm
 		return nil
 	}
 	msg := fmt.Sprintf("Updating schema of database %s with foreign key constraints ...", dbURI)
```

Comment:
Should we add some sort of a warning message before updating foreign keys that says creation can be slow due to high number of foreign keys?